ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml

```diff
@@ -170,11 +170,11 @@ tests:
     architecture: amd64
     cloud: aws
     labels:
-      region: us-east-1
+      region: us-east-2
```

Comment:
We don't have any pools in us-east-2 btw.   Not sure the impact of changing this label

Reply:
I'm wrong.  nvm, 4.16 is in us-east-1.   disregard